backend/entities.json

```diff
@@ -0,0 +1,127 @@
+{
+  "airline": {
+    "id": "1234567890",
+    "name": "",
+    "createdDate": "",
```

Comment:
Could you use 'createdAt' and 'updatedAt' fields instead of 'createdDate' and 'updatedDate'? (https://mongoosejs.com/docs/timestamps.html)

![image](https://user-images.githubusercontent.com/20149509/171230116-168afb5e-8b23-427a-8d24-cbc68464e2c1.png)

Reply:
Yes, sure. I created this file before I knew about timestamps